apps/juxtaposition-ui/src/models/logs.js

```diff
@@ -0,0 +1,35 @@
+const { Schema, model } = require('mongoose');
+
+const actionEnum = [
+	'REMOVE_POST',
+	'IGNORE_REPORT',
+	'LIMIT_POSTING',
+	'TEMP_BAN',
+	'PERMA_BAN',
+	'UNBAN',
+	'UPDATE_USER',
+	'MAKE_COMMUNITY',
+	'UPDATE_COMMUNITY',
+	'DELETE_COMMUNITY'
+];
+
+const auditLogSchema = new Schema({
+	actor: Number,
```

Comment:
Should consider making these like:
```js
actor: {
  type: Number,
  required: true,
}
```
That will make life a bit easier once TypeScript gets involved later, since these properties won't be marked optional